src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/interfaces/AnnotationGroup.java

```diff
@@ -0,0 +1,10 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator.interfaces;
+
+/**
+ * An annotation group is a set of annotation that have something in common and should be added at the same time.
+ * This enum lists the known groups.
+ */
+public enum AnnotationGroup{
+    Standard,
+    ActiveRegionBased
```

Comment:
Are we getting rid of UG?  There is a set of StandardUGannotations in GATK3 right now that aren't used for HC.